apps-maven-builder/build-quarkus-app.sh

```diff
@@ -42,7 +42,7 @@ maven_plugins_gav=("org.apache.maven.plugins:maven-resources-plugin:3.3.1" "org.
 properties_with_versions=("compiler-plugin.version:3.11.0" "surefire-plugin.version:3.1.2")
 
 # extra dependencies required for offline mode
-quarkus_extensions_extra_deps="com.aayushatharva.brotli4j:native-linux-aarch64:1.16.0.redhat-00005"
```

Comment:
I'm moving this to upstream version, because in next PR I will parametrize this version (it will be defined in the pom) and it will be changed from upstream version to productized version in productization process so we dont miss upgrading it when Quarkus prod is upgraded.